_episodes/08-ghpages-intro.md

```diff
@@ -16,6 +16,32 @@ GitHub Pages is a static site hosting service that hosts your website directly f
 
 [Jekyll](https://jekyllrb.com/) is used to generate your site from plain text to a static website, and [GitHub Pages](https://pages.github.com/) hosts the generated website. There are several [Jekyll Themes](http://jekyllthemes.org/) that you can choose from. For today, we will be working with the [Jekyll Now](http://www.jekyllnow.com/) theme designed by Barry Clark. It's a really quick and easy way to get up and running with a personal website and blog. 
 
-The first element of creating your website from a template is forking the template repository to your own GitHub repository, and giving it a specific name so that GitHub pages knows where to build the repository to. Go to the [Jekyll Now GitHub repository](https://github.com/barryclark/jekyll-now). Fork to your own repository by following these steps: 
+The first element of creating your website from a template is forking the template repository to your own GitHub repository, and giving it a specific name so that GitHub pages knows where to build the repository to. Go to the [Jekyll Now GitHub repository](https://github.com/barryclark/jekyll-now). Get started by following these steps:
 
-[!]../fig/fork.png
\ No newline at end of file
+**1. Fork the template GitHub repository to your own repository.** 
+
+![](../fig/fork.png)
+
+**2.  Go to the settings page of your forked repository.**
```

Comment:
Next we will go to the settings page in our version of the forked repo in order to change the name of the repository.  As is we would have to make modifications to have it render at __yourusername.github.io/jekyll-now__ but we actually want it to be the main webpage so we will change the name of the repo to __yourusername.github.io__